plugins/@grouparoo/demo/src/util/env.ts

```diff
@@ -12,6 +12,14 @@ const ENV_VARIABLES = {
     "GOOGLE_SERVICE_PRIVATE_KEY",
   ],
   mailchimp: ["MAILCHIMP_DESTINATION_LIST_ID", "MAILCHIMP_API_KEY"],
+  redshift: [
+    "REDSHIFT_PASSWORD",
+    "REDSHIFT_USER",
+    "REDSHIFT_PORT",
+    "REDSHIFT_HOST",
+    "REDSHIFT_SCHEMA",
+    "REDSHIFT_DATABASE",
+  ],
```

Comment:
I've been going with `REDSHIFT_URL` (see 1password)

Reply:
The parsing seemed to be more trouble that it was worth, so I made it more like the other data warehouses.